### PR TITLE
Basic load from multielasticsearch

### PR DESCRIPTION
**Pull request by cggaurav**

### CODE REVIEW

**Comment by cggaurav** [comment]
@evantahler Let me know how you want me to proceed. I am going with index and dump as commands.

**Comment by cggaurav** [comment]
What's the best practice to pass other options like --maxSockets to elasticdump from multielasticdump?

**Comment by evantahler** [comment]
I would add `--maxSockets` in to this in a separate pull request.  That will be simpler to merge and easier to review

**Comment by cggaurav** [comment]
@evantahler Any reason why just this is failing? https://travis-ci.org/taskrabbit/elasticsearch-dump/jobs/131411753

File: bin/multielasticdump [hunk 1]
```diff
@@ -46,6 +54,12 @@ for(var i in defaults){
   }
 }
 
+options.action = defaults.action;
+
+if(argv._[0]) {
+  options.action = argv._[0];
+}
```
Comment (evantahler) [inline bin/multielasticdump]: This shouldn't be needed?

**Comment by evantahler** [comment]
This is a really good start!

I don't think "action" is really the right variable name here.... perhaps `direction`?  `dump` and `load` are the names (index itself is also confusing, something ES acknowledges).  This would then be `--direction=dump` and `direction=load`.

You should also add documentation about the feature in the same PR (add it to the readme)

**Comment by cggaurav** [comment]
Do you think we should exit if we don't find a mapper or analyzer file while loading? Generally, the .kibana index doesn't have that, where we can continue with a warning?

**Comment by cggaurav** [comment]
The test failing for 
```
1) parent child ES to ES dump should maintain parent-child relationships the dump transfered:
      Uncaught AssertionError: expected 0 to be 24
      + expected - actual
      -0
      +24
 ``` 
is something else based on the test data, not for the PR, I think.

**Comment by cggaurav** [comment]
@evantahler Any ideas why https://travis-ci.org/taskrabbit/elasticsearch-dump/builds/134174947 is failing?

**Comment by cggaurav** [comment]
I had to increase the timer for clearing indices @evantahler

**Comment by cggaurav** [comment]
@evantahler Want to get you another PR with bulk and maxSockets as another PR - let me know what you need here for merge.

**Comment by evantahler** [comment]
This look great!

I'll merge this in now, but you need to add tests in another pull request.

**Comment by evantahler** [comment]
https://github.com/taskrabbit/elasticsearch-dump/releases/tag/v2.2.0 released!

**Pull request merged**